Tests/RequestDLTests/Internals/Extensions/XCTestCase+.swift

```diff
@@ -2,12 +2,11 @@
  See LICENSE for this package's licensing information.
 */
 
-#if swift(<5.8)
 import XCTest
 
 extension XCTestCase {
 
-    func fulfillment(
+    func _fulfillment(
```

Comment:
qual seria a razão para renomear essa função?

Reply:
The fulfillment method with async/await support was implemented in Xcode 14.3. However, currently, we are using Xcode 14.2 in our test workflows.

Apparently, this XCTest update is not available on Ubuntu and is not tied to a specific version of Swift.

I can do more research to better understand it.

So, I added the "_" to indicate that it is an internal method of RequestDL without losing the context related to the encapsulated method.